h/templates/activity/search.html.jinja2

```diff
@@ -173,6 +173,44 @@
 </section>
 {% endmacro %}
 
+{% macro group_users(title, users, creator) %}
+<section class="search-result-sidebar__section">
+  <h3 class="search-result-sidebar__subtitle">
+    {% trans %} {{title}} {% endtrans %}
+    <span class="search-result-sidebar-title__annotations-count">
+      {{ users|length }}
+    <span>
+  </h3>
+  <ul>
+    {% for user in users %}
+      <li>
+        <button type="submit"
+                form="search-bar"
+                name="toggle_user_facet"
+                value="{{ user.userid }}"
+                {% if user.faceted_by %}
+                  title="{% trans username=user.username %}Remove {{ username }} from the search query{% endtrans %}"
+                {% else %}
+                  title="{% trans username=user.username %}Limit the search to annotations by {{ username }}{% endtrans %}"
+                {% endif %}
+                class="search-result-sidebar__username">
+          {{ user.username }}
+          <span class="search-result-sidebar__annotations-count">
+            {{ user.count }}
+          </span>
+          {% if user.userid == creator %}
+            <span class="search-result-sidebar__annotations-count">
+              creator
```

Comment:
In the mock, the annotation count and "creator" label use different fonts to differentiate them:

<img width="167" alt="screenshot 2018-02-13 12 08 37" src="https://user-images.githubusercontent.com/2458/36149217-d676ef94-10b6-11e8-93ee-50a39bdc0797.png">

In the implementation, the two have the same font and wasn't immediately clear to me that "creator" was a separate label:

<img width="316" alt="screenshot 2018-02-13 12 08 08" src="https://user-images.githubusercontent.com/2458/36149238-e7b6199c-10b6-11e8-8dd0-ed3a04ae7b18.png">

I would rather not change the style of the annotation count in this PR so let's leave as-is but I think we'll want to change this.

Reply:
Ah, OK, that's fine then.